content/departments/product-engineering/engineering/process/cloud_processes.md

```diff
@@ -0,0 +1,43 @@
+# Sourcegraph Cloud Processes
+
+## Closing down an organization on Sourcegraph Cloud
+
+During the open-beta, customers receive 30 days to try out Sourcegraph Cloud with Organizations. Once this period has passed, customers need to reach out in order to continue leveraging Sourcegraph Cloud.
+
+The following process documents the queries that need to be executed in order to completely remove an organization if a customer does not reach out.
+
+### Queries
```

Comment:
Hmm, don't we just need to hit the org hard delete GraphQL endpoint and the no need to have direct access to the DB? (since this section it basically describing what the hard delete endpoint does exactly in my understanding).

Reply:
@unknwon from what I understand this is for support which often does things with access to the DB. We might have to speak to @ryphil when he's back just to make sure what it is he had in mind

Reply:
> @unknwon from what I understand this is for support which often does things with access to the DB. We might have to speak to @ryphil when he's back just to make sure what it is he had in mind

If we have no other ways, I'd agree we do need access to DB, but if we have equivalent and less error-prone ways, I'd suggest avoid direct access to DB 😁

Reply:
@pjlast or @unknwon - I'd also suggest avoiding direct DB access. 

Regardless, given then news that we won't be onboarding many new teams, we shouldn't need to run this process very often. And rather than hand off to support, our team would handle the request. Thus, i'd say whatever is easiest to finish this task up quickly.